public/js/actions/subscriptions.js

```diff
@@ -33,43 +33,68 @@ export function getUserSubscriptions(jquery=$) {
 }
 
 
-export function createSubscription(braintreeToken, productId,
-                                   creditCard, jquery=$,
+export function createSubscription(productId, payMethod,
+                                   braintreeToken, jquery=$,
                                    BraintreeClient=braintree.api.Client) {
   return dispatch => {
 
-    var client = new BraintreeClient({
-      clientToken: braintreeToken,
-    });
+    function requestSub(opts={}) {
```

Comment:
It looks like this isn't optional so maybe just `requestSub(payMethodUri)` ? Or if you want to pass it as a parameter for explicitness it could be `requestSub({payMethodUri})` I think.